examples/with-cloudinary/.env.local.example

```diff
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME=
+CLOUDINARY_API_KEY=
+CLOUDINARY_API_SECRET=
+CLOUDINARY_FOLDER=
```

Comment:
What is the standard operating procedure with apps that require environment variables in order to be functional? Is simply having this file enough or does it need to be called out in the readme?

Reply:
Wouldn't hurt to also call it out in the README. Will do this!